variables.tf

```diff
@@ -158,6 +168,11 @@ variable "send_job_alerts_daily_email_command" {
   default     = ["rake", "verbose", "daily_emails:send"]
 }
 
+variable "backfill_jobseeker_alert_data_command" {
+  description = "The Entrypoint for the backfill_jobseeker_alert_data task"
+  default     = ["rake", "verbose", "jobseeker_alerts:statistics:backfill:alert_data"]
+}
```

Comment:
We also need to run this after merging. Did we confirm that it works on staging?

Reply:
Nope. We probably need to do a staging deploy and check that out 👍

Reply:
I think we should always be doing those as soon as we merge to `develop`.